antenna/throttler.py

```diff
@@ -230,6 +230,51 @@ def match_b2g(throttler, data):
     return False
 
 
+def match_seamonkey_gt_2_49_5(throttler, data):
+    """Reject SeaMonkey > 2.49.5 (bug #1604848).
+
+    NOTE(willkg): Remove this January 2021.
+
+    """
+    product = data.get("ProductName", "").lower().strip()
+    version = data.get("Version", "")
+    channel = data.get("ReleaseChannel", "")
+
+    if product != "seamonkey" or not version or channel != "release":
+        return False
+
+    try:
+        version = tuple(int(n) for n in version.split("."))
```

Comment:
I'm not sure what happens when the version number is something like `2.70a1`, but I'm guessing `ReleaseChannel` is set to something other than `"release"`.

Reply:
I don't think there's a case where the release channel is "release" and the version has non-digits in it. However, we get junk data periodically, so if we had a junk version, it'd raise a `ValueError` and that'll cause this function to return `False` which is the right thing to do.

That variation should be covered by a test, but it's not. I'll add that now.